<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigWidget</class>
 <widget class="QWidget" name="ConfigWidget">
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_onTop">
     <property name="text">
      <string>Always on top:</string>
     </property>
     <property name="buddy">
      <cstring>checkBox_onTop</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QCheckBox" name="checkBox_onTop">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_center">
     <property name="text">
      <string>Center on show:</string>
     </property>
     <property name="buddy">
      <cstring>checkBox_center</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QCheckBox" name="checkBox_center">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_followMouse">
     <property name="text">
      <string>Show on current screen:</string>
     </property>
     <property name="buddy">
      <cstring>checkBox_followMouse</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QCheckBox" name="checkBox_followMouse">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_hideOnFocusOut">
     <property name="text">
      <string>Hide on focus out:</string>
     </property>
     <property name="buddy">
      <cstring>checkBox_hideOnFocusOut</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QCheckBox" name="checkBox_hideOnFocusOut">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_hideOnClose">
     <property name="text">
      <string>Hide on close:</string>
     </property>
     <property name="buddy">
      <cstring>checkBox_hideOnClose</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QCheckBox" name="checkBox_hideOnClose"/>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_clearOnHide">
     <property name="text">
      <string>Clear input on hide:</string>
     </property>
     <property name="buddy">
      <cstring>checkBox_clearOnHide</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QCheckBox" name="checkBox_clearOnHide"/>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_systemShadow">
     <property name="text">
      <string>Use system shadow:</string>
     </property>
     <property name="buddy">
      <cstring>checkBox_systemShadow</cstring>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QCheckBox" name="checkBox_systemShadow">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>checkBox_onTop</tabstop>
  <tabstop>checkBox_center</tabstop>
  <tabstop>checkBox_followMouse</tabstop>
  <tabstop>checkBox_hideOnFocusOut</tabstop>
  <tabstop>checkBox_hideOnClose</tabstop>
  <tabstop>checkBox_clearOnHide</tabstop>
  <tabstop>checkBox_systemShadow</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>checkBox_center</sender>
   <signal>toggled(bool)</signal>
   <receiver>checkBox_followMouse</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>187</x>
     <y>86</y>
    </hint>
    <hint type="destinationlabel">
     <x>187</x>
     <y>107</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_center</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_followMouse</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>210</x>
     <y>83</y>
    </hint>
    <hint type="destinationlabel">
     <x>147</x>
     <y>105</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
